ScibConfig.py

```diff
@@ -75,16 +75,18 @@ def get_from_method(self, method, key):
         if key not in self.METHODS[method]:
             return False
             # raise ValueError(f"{key} not a valid attribute of scenario {scenario}")
-        return self.METHODS[method][key]
+        value = self.METHODS[method][key]
+        if key == 'output_type':
+            return value if isinstance(value, list) else [value]
+        return value
 
-    def get_hvg(self, wildcards, adata_path):
+    def get_hvg(self, wildcards, output_pattern, **kwargs):
         n_hvgs = self.get_feature_selection(wildcards.hvg)
         if n_hvgs == 0:
             return ""
         if self.get_from_method(wildcards.method, "R"):
-            path_parts = adata_path.split('.')
-            path_parts[-2] += '_hvg'
-            hvg_path = '.'.join(path_parts)
+            p = Path(expand(output_pattern, **wildcards, **kwargs)[0])
+            hvg_path = (p.parent / f'{p.stem}_hvg').with_suffix(p.suffix)
```

Comment:
this is really cool... didn't know these operators were overloaded for Path variables.